client/reader/start/main.jsx

```diff
@@ -1,20 +1,43 @@
 // External dependencies
```

Comment:
OK, I finally actually read all of the components doc.

> Components are reusable in nature, regardless of where they are placed in the directory structure. This is an important clarification, since we don't place components in folders based on whether they are reusable or not. It's not a requirement for a component to live in client/components to be reused. Indeed, we use the my-sites/site component to render the current site in the editor, to render the site picker in the sidebar, to render sites in "Me", etc. The pieces that go to client/components are those that don't have a natural preference to one of our main section groups and are thus purer UI building blocks.

That's ... confusing. Section specific components should be in their sections and css rules should be scoped by section. Generic components should be in `components` and should not have a scope. The "components are generic no matter where they are" defeats the purpose of having folders in the first place. Why doesn't everything live in `components`?